packages/backend/src/services/UserService.test.ts

```diff
@@ -353,7 +343,14 @@ describe('UserService', () => {
             );
         });
         test('should link openid to an existing user that has another OIDC with the same email', async () => {
-            await userService.loginWithOpenId(openIdUser, undefined, undefined);
+            const service = createUserService({
+                ...lightdashConfigMock,
+                auth: {
+                    ...lightdashConfigMock.auth,
+                    enableOidcLinking: true,
```

Comment:
do we need another test to check what happens if this is false ?

Reply:
`should create user` covers that case as both use `await service.loginWithOpenId(openIdUser, undefined, undefined);` but this once has the feature flag enabled.